cmd/os_mgmt/port/zephyr/src/zephyr_os_mgmt.c

```diff
@@ -63,12 +64,21 @@ os_mgmt_impl_task_info(int idx, struct os_mgmt_task_info *out_info)
     }
 
     *out_info = (struct os_mgmt_task_info){ 0 };
+
+#ifdef CONFIG_THREAD_NAME
+    strncpy(out_info->oti_name, thread->name, sizeof out_info->oti_name);
```

Comment:
What if Zephyr's `CONFIG_THREAD_MAX_NAME_LEN` is configured with a value larger than `OS_MGMT_TASK_NAME_LEN`, wouldn't this end up with a missing NULL terminator?

Reply:
Thanks for your review.
Indeed oti_name maybe overflow, I modify it, Do you think it is appropriate to modify this way.

Reply:
That works, but you could just add the NULL after finishing the `strncpy`:

```
out_info->oti_name[OS_MGMT_TASK_NAME_LEN - 1] = '\0';
```

It would avoid doing an extra write of the whole the buffer (which `strncpy` also does!).

Reply:
Good suggestion，Thanks